CONTRIBUTING.md

```diff
@@ -125,24 +125,30 @@ The process of proposing a change to Talawa Admin can be summarized as:
       1. _General Information_
          1. The current code coverage of the repo is: [![codecov](https://codecov.io/gh/PalisadoesFoundation/talawa-admin/branch/develop/graph/badge.svg?token=II0R0RREES)](https://codecov.io/gh/PalisadoesFoundation/talawa-admin)
          2. You can determine the percentage test coverage of your code by running these two commands in sequence:
+
             ```
             npm install
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project